notify/msteams/msteams.go

```diff
@@ -35,9 +35,15 @@ import (
 )
 
 const (
-	colorRed   = "8C1A1A"
-	colorGreen = "2DC72D"
-	colorGrey  = "808080"
+	ColorAlertFiring   = "#D63232"
```

Comment:
I think `ColorAlertFiring` and `ColorAlertResolved` can be deleted as not used?

Reply:
Ah! Good catch, I removed the need for these constants right before submitting the code but forgot to remove the constants 😅

Reply:
Removed: https://github.com/grafana/prometheus-alertmanager/pull/90/commits/bf5d7a3cf1725ce45e845fd4794b07fa006d2f3e